gnmi/gnmi.go

```diff
@@ -277,7 +292,7 @@ func (s *Server) Set(ctx context.Context, req *gpb.SetRequest) (*gpb.SetResponse
 
 		// TODO(wenbli): Reject paths that try to modify read-only values.
 		// TODO(wenbli): Question: what to do if there are operational-state values in a container that is specified to be replaced or deleted?
-		err := set(s.configSchema, s.c.cache, s.c.name, req, true)
+		err := set(s.configSchema, s.c.cache, s.c.name, req, true, s.validators)
 
 		// TODO(wenbli): Currently the SetResponse is not filled.
 		return &gpb.SetResponse{
```

Comment:
It looks like the status.Error() is missing here.

Reply:
everything returned by set is status.Error so we don't want to wrap it twice